src/modules/user/user.controller.ts

```diff
@@ -150,4 +150,24 @@ export class UserController {
   refreshToken(@UserDecorator() user: User) {
     return this.userUseCases.getAuthTokens(user);
   }
+
+  @Post('/search')
+  @HttpCode(200)
+  @ApiOperation({
+    summary: 'Search items by name',
+  })
+  async searchItemsByName(
+    @UserDecorator() user: User,
+    @Body() body: { plain_name: string },
```

Comment:
use camelCase, as this is a new endpoint and we do not depend on the clients here.

Reply:
Also, this is not a POST. You are getting, and the search param should be a query param.